geekhours.md

```diff
@@ -0,0 +1,97 @@
+# Geek Hours Command Examples
+
+## Help messages
+```
+$ geekhours -h
+usage: geekhours [-h] {course,done} ...
+
+Geekhours is a management tool for the study time.
```

Comment:
```
geekhours is a simple study time management tool.
```
You can use the same sentence of the README.md

Reply:
Fixed.